djangocms_alias/templates/djangocms_alias/alias_list.html

```diff
@@ -9,6 +9,15 @@ <h2 class="cms-aliases-page-heading" id="{{ category.name|slugify }}">
             </span>
             <a href="../.." class="cms-aliases-page-back cms-alias-btn">{% trans "Back" %}</a>
         </h2>
+        <div class="cms-alias-site-filter">
+            <label class="inline">{% trans "Filter by Site" %}</label>
+            <select id="sitefilter" class="small-2">
+                <option value='all' >{% trans "ALL" %}</option>
```

Comment:
I don't think that we need all, the default if no option is selected is to show all of the results. This option will just be a standard Django "----" entry with no value set. If this is changed and no value is set the get param is removed.

Reply:
i did keep ---- initially. but replaced with 'ALL' later. will update back to '-----'